backend/src/main/java/com/bakdata/conquery/models/query/filter/event/number/MoneyFilterNode.java

```diff
@@ -6,9 +6,9 @@
 import lombok.ToString;
 
 @ToString(callSuper = true)
-public class MoneyFilterNode extends NumberFilterNode<Range.LongRange> {
+public class MoneyFilterNode extends NumberFilterNode<Range.MoneyRange> {
 
-	public MoneyFilterNode(Column column, Range.LongRange filterValue) {
+	public MoneyFilterNode(Column column, Range.MoneyRange filterValue) {
```

Comment:
Hat MoneyRange Auswirkungen auf das Frontend, oder braucht das eine Storagemigration?

Reply:
Das ist so implementiert, dass es immer noch "cent" Beträge vom FE erwartet und die dann mapped in BigDecimal, ist also kompatibel mit der vorherigen API/Schema